sunpy/net/jsoc/tests/test_jsoc.py

```diff
@@ -40,6 +40,15 @@ def test_empty_jsoc_response():
     assert len(Jresp) == 0
 
 
+@pytest.mark.remote_data
+def test_return_query_args():
+    res = client.search(a.jsoc.PrimeKey('HARPNUM', 3604),
+                        a.jsoc.Series('hmi.sharp_cea_720s'),
+                        a.jsoc.Segment('Bp') & a.jsoc.Segment('magnetogram'),
+                        a.jsoc.Notify('sunpy@sunpy.net'))
```

Comment:
As we aren't doing an export call here we don't need an email.
```suggestion
                        )
```

Reply:
:+1: